index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James' Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+    <script src="./script.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200..700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <nav class="nav">
+        <div class="nav__img">
+            <img src="./assests/img/profile-pic.png" alt="Picture of James">
+        </div>
+        <div id = "navbar" class="nav__desktop">
+            <a href="#">Home</a>
+            <a href="#about">About</a>
+            <a href="#skills">Skills</a>
+            <a href="#projects">Projects</a>
+            <a href="#contact">Contact</a>
+        </div>
+        <div id = "hamburger" class = "nav__hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </nav>
+    </nav>
+    <header id="hero" class="hero">
+        <h1>Hey, my name is James Nemeth!</h1>
+        <h2>Junior Full-Stack Developer</h2>
+    </header>
+    <section id="about" class="about">
+        <h2>Few things about me are 
+            <span class="line__container">
+                <span id ="line-one" class="line">_</span>
+                <span id ="line-two" class="line">_</span>
+                <span id ="line-three" class="line">_</span>
+            </span>
+        </h2>
+        <article class="about__container">
+            <div class="about__experience">
+                <h3>Experience</h3>
+                <p>1 yr Frontend </p>
+                <p>1 yr Backend</p>
+            </div>
```

Comment:
Do you have work experience here to back this up? If not I'd probably pull it out.